### PR TITLE
add getmin3nums function

### PR DESCRIPTION
**Pull request by alaaeldein108**

### CODE REVIEW

File: SWEgit/src/com/company/Main.java [hunk 1]
```diff
@@ -198,4 +198,25 @@ public static void GetAverage(String[] arr) {
         System.out.println("Average : "+sum/arr.length);
         }
     }
+    public static void getmin3nums(int[]arr)
```
Comment (EhabMostafa) [inline SWEgit/src/com/company/Main.java]: The input will be string array in the main so i suggest to edit your function to be string array and covert this array to integer

File: SWEgit/src/com/company/Main.java [hunk 2]
```diff
@@ -198,4 +198,25 @@ public static void GetAverage(String[] arr) {
...
+	{
+		Arrays.sort(arr);
+		int[]temp=new int [arr.length];
```
Comment (EhabMostafa) [inline SWEgit/src/com/company/Main.java]: Take temp of array before sort it

**Review by EhabMostafa — COMMENTED** [review]
Please call you function in case 18 (Excute all)

**Review by abdallahsayed98 — APPROVED** [review]

**Review by abdallahsayed98 — APPROVED** [review]

**Review by EhabMostafa — CHANGES_REQUESTED** [review]
Call you function in case 18

**Review by EhabMostafa — APPROVED** [review]

**Pull request merged**